s3/aide.go

```diff
@@ -94,3 +94,34 @@ func (svc *Service) SetACL(v *string) {
 func (svc *Service) SetSSE(v *string) {
 	svc.sse = v
 }
+
+// ListObjectsKeysV2Pages will list the bucket keys page-wise
```

Comment:
Please include a period at the end of the first sentence of method documentation.

Reply:
Done.